api/middleware/validation.js

```diff
@@ -17,3 +18,21 @@ exports.validateAndSanitizeIncidentQueries = async (req, _res, next) => {
     next({ status: 400, message: err.message });
   }
 };
+
+exports.validateAndSanitizeIncidentObject = async (req, _res, next) => {
+  if (!req.body) {
+    return next({ status: 400, message: 'incident object required' });
+  }
+
+  try {
+    const validIncident = await incidentObject.validate(req.body, {
+      stripUnknown: true,
```

Comment:
stripUnknown will strip the date_created if not in the validation schema

Reply:
Great catch! Thanks!

Reply:
Although, ideally that data should be immutable so it should not be available as a field to edit. And since the model will only update the columns passed in, stripping unknown seems like the best bet. What do you think?

Reply:
Hmm, but you're right, because of the admin's need to manually POST a new incident. Thanks!